src/helm/benchmark/scenarios/commonsense_scenario.py

```diff
@@ -17,6 +17,103 @@
 )
 
 
+_SPLIT_TRANSLATION = {
+    "train": TRAIN_SPLIT,
+    "val": VALID_SPLIT,
+    "test": TEST_SPLIT,
+}
+
+
+def _make_instance(question: str, answers: List[str], correct_answer: str, split: str):
+    references = []
+    for answer in answers:
+        references.append(Reference(Output(text=answer), tags=[CORRECT_TAG] if answer == correct_answer else []))
+    return Instance(
+        input=Input(text=question),
+        references=references,
+        split=_SPLIT_TRANSLATION[split],
+    )
+
+
+class HellaswagScenario(Scenario):
```

Comment:
I'd suggest moving this into their own individual module files, but that can be another PR.